config/defaults.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  taskcluster: {
+    queueBaseUrl: 'https://queue.taskcluster.net/v1',
+    authBaseUrl: 'https://auth.taskcluster.net/v1',
+    credentials: {
+      clientId:     undefined,
+      accessToken:  undefined,
+    }
+  },
+  pulse: {
+    username: undefined,
+    password: undefined,
+  },
+  provisioner: {
+    id:                   'aws-provisioner2',
+    workerTypeTableName:  'AwsWorkerTypes',
+    publishMetaData:      'false',
+    statsComponent:       'aws-provisioner2',
+    awsKeyPrefix:         'aws-provisioner2-managed:',
+    iterationInterval:    1000 * 10,
+    awsInstancePubkey:    undefined,
+    allowedRegions:       'us-west-1', // comma seperated list
+  },
+  server: {
+    publicUrl:  'https://localhost',
+    port:       5556,
+    env:        'development',
+    forceSSL:   false,
+    trustProxy: false,
+  },
+  azure: {
+    accountName: undefined,
+    accountKey:  undefined,
```

Comment:
You don't need this anymore... You need it for your manage tables utility... but you really should use the API end-points on auth for getting temporary azure credentials for a specific table..

Take a look at how this done in the index:
https://github.com/taskcluster/taskcluster-index/blob/master/bin/server.js#L37-L42
See `auth.azureTableSAS` for details on scopes: http://docs.taskcluster.net/auth/api-docs/#azureTableSAS